src/app/projects/states/portfolio/directives/portfolio-welcome-step/portfolio-welcome-step.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, Input, Inject, OnInit } from '@angular/core';
+import { DoubtfireConstants } from 'src/app/config/constants/doubtfire-constants';
+
+@Component({
+  selector: 'portfolio-welcome-step',
+  templateUrl: 'portfolio-welcome-step.component.html',
+})
+export class PortfolioWelcomeStepComponent {
+  @Input() advanceActiveTabs: any;
+  constructor(private constants: DoubtfireConstants) { }
+  public externalName: any;
+
+  public advanceActiveTab(input: number) {
+    return this.advanceActiveTabs(input)
```

Comment:
I wonder if there's a better way to write this, as this function seems to only ever receive and pass the value `1` to progress to the next step. Does the html file even need to pass the value to this function? What do you think @RayGuo-ergou

Reply:
> I wonder if there's a better way to write this, as this function seems to only ever receive and pass the value `1` to progress to the next step. Does the html file even need to pass the value to this function? What do you think @RayGuo-ergou

You are right we can use 
```javascript
public advanceActiveTab() {
    return this.advanceActiveTabs(1)
}
```